src/core/math/Matrix.js

```diff
@@ -363,7 +351,7 @@ export default class Matrix
 
         const delta = Math.abs(skewX + skewY);
 
-        if (delta < 0.00001)
+        if (delta < 0.00001 || Math.abs((2 * Math.PI) - delta) < 0.00001)
```

Comment:
Minor thing, could you use import `PI_2` from **src/core/const.js** instead of using `2 * Math.PI`?